packages/common/lib/theme.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart';
+
+part 'theme/scheme.dart';
+part 'theme/extension.dart';
+part 'theme/widgets/button.theme.dart';
+part 'theme/widgets/input.theme.dart';
+part 'theme/widgets/scheme.theme.dart';
+part 'theme/widgets/text.theme.dart';
+
+@WidgetbookTheme(name: 'Light')
```

Comment:
We can remove this and simply mark it as WidgetbookTheme inside the Widgetbook Package, I'm trying to setup the base package for Widgetbook now will make a PR.